recipes/ipyscales/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "ipyscales" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "8d77d4b29ccbab9c0a45a4794406e6d66812707efc26ce32f86cb3d64ad51128" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - ipywidgets >=7,<8
+
+test:
+  imports:
+    - {{ name }}
+
+about:
+  home: http://github.com/vidartf/ipywdatawidgets
```

Comment:
Is this right?

Reply:
Woops, left over from copy-paste of another recipe in progress. Fixed now.